firefox_puppeteer/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Permissions(BaseLib):
+    def remove(self, host, permission):
```

Comment:
This needs documentation of the parameters, and a unit test. So maybe it would be wiser to file a separate bug for its implementation.